bciers/libs/components/src/auth/SessionTimeoutHandler.tsx

```diff
@@ -30,11 +30,19 @@ const SessionTimeoutHandler: React.FC = () => {
       .catch((error) => console.error("Failed to fetch logout URL:", error));
   }, []);
 
-  const handleLogout = () => signOut({ callbackUrl: logoutUrl });
```

Comment:
If there's no `logoutUrl`, then the fallback is the current page (see the `signOut` function in `bciers/node_modules/next-auth/src/react.tsx`). So when a user isn't being redirected, possibly what's happening is that something went wrong with `getEnvValue` in the `useEffect`, and we don't actually have a `logoutUrl`. I couldn't trigger this locally to check, unfortunately.

If we don't get the `logoutUrl`, we could:
- redirect the user to a different page (maybe make a fallback one that just says "You've been logged out" or something)
- throw an error